app/src/main/java/ch/epfl/sdp/contamination/Layman.java

```diff
@@ -31,9 +43,23 @@ public Layman(InfectionStatus initialStatus, String uniqueID){
     }
 
     public Layman(InfectionStatus initialStatus, float infectedWithProbability, String uniqueID) {
-        this.myStatus = initialStatus;
-        this.infectedWithProbability = infectedWithProbability;
+        DateFormat format = new SimpleDateFormat("E MMM dd hh:mm:ss zzz yyyy");
+
+        this.infectionHistory = new ConcreteManager<>(
+                CoronaGame.getContext(),
+                uniqueID + ".csv",
```

Comment:
Since the infectionHistory is stored locally foreach user, does it make sense to name the csv file after the uniqueUserID ?

Reply:
It's not mandatory, agree. Just to avoid future conflicts